pykechain/models/activity2.py

```diff
@@ -462,6 +464,20 @@ def download_as_pdf(self, target_dir=None, pdf_filename=None, paper_size=PaperSi
         if response.status_code != requests.codes.ok:  # pragma: no cover
             raise APIError("Could not download PDF of activity {}".format(self.name))
 
+        # If appendices are included, the request becomes asynchronous
+        if include_appendices:
+            data = response.json()
+
+            # Download the pdf async
+            url = urljoin(self._client.api_root, data['download_url'])
+
+            while True:
+                response = self._client._request('GET', url=url)
+
+                if response.status_code == requests.codes.ok:  # pragma: no cover
+                    break
+                time.sleep(2)
```

Comment:
Make the two seconds a top level constant
Also make a timeout of 60 seconds or something.  Make the timeout also a constant or function variable.